js-core-1/week-3/lesson.md

```diff
@@ -364,19 +363,11 @@ names.forEach(function (name, index) {
 });
 ```
 
-This logs each name to the console as hoped, but we notice that the names are not formatted correctly. You might be tempted to format the name inside of the `forEach` function.
-
-However, it is good practise to write small functions with a single responsibility. So instead, we can write a `formatName` function (which we can re-use in other places) and pass it to `.map()` before calling `.forEach()`.
-
-```js
-function formatName(name) {
-  return name.split("")[0].toUpperCase() + name.slice(1);
-}
+### Exercise (10 minutes)
 
-names.map(formatName).forEach(function (name, index) {
-  console.log(index + ": " + name);
-});
-```
+- Create a function that takes a `birthYear`, and returns the age of someone
+- With an array of ages, create another array filled with their birth years `[ 1964, 2008, 1999, 2005, 1978, 1985, 1919 ]`
+- `console.log` the birth years array
```

Comment:
I remember this exercise being kind of confusing in class. I _think_ this line is the wrong way around - it's meant to be saying:

```suggestion
- With an array of birth years `[ 1964, 2008, 1999, 2005, 1978, 1985, 1919 ]`, create another array filled with their ages.
- `console.log` the ages array
```

Reply:
Could we make this even clearer:

> You are given an array with year that 7 people were born, `[ 1964, 2008, 1999, 2005, 1978, 1985, 1919 ]`. Take this array and create another array containing their ages.